.github/workflows/python-client.yml

```diff
@@ -0,0 +1,48 @@
+name: Python client checks, package build and deployment
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - develop
+      - master
+    paths:
+      - ./client/
```

Comment:
`python_client` ?

Reply:
For now, the python client is stored in the `./client` directory,